### ISSUE TITLE
default to guess_dates = FALSE in clean_data

### ISSUE THREAD

**Issue by thibautjombart** [reconhub/linelist#102]
From what I have seen of using `clean_data`: as soon as the data starts having a realistic size, things become quite messy and looking for dates in all variables is a big timewaste, the common practice becoming something along the lines of:

```r

x_clean <- x_raw %>%
    clean_data(guess_dates = FALSE) %>%
    mutate_at(contains("date"), guess_dates)

```

I think it would make sense to default to `guess_dates = FALSE` in `clean_data`. Will implement this change via PR.

**Comment by zkamvar** [comment]
Should this be closed? I think it was taken care of in #103

**Comment by thibautjombart** [comment]
Yep thanks
